client/packages/system/src/Encounter/DetailView/AppBarButtons.tsx

```diff
@@ -17,7 +17,7 @@ export const AppBarButtonsComponent: FC<AppBarButtonsProp> = ({
 
   return (
     <AppBarButtonsPortal>
-      <Grid container gap={1}>
+      <Grid container gap={1} flexWrap="nowrap">
```

Comment:
Fixes this

![Screenshot 2025-02-04 at 4 26 46 PM](https://github.com/user-attachments/assets/480a301a-fafc-498a-bbe1-bf2f62a92a32)